enterprise/ha/src/test/java/org/neo4j/kernel/ha/cluster/SwitchToSlaveCopyThenBranchTest.java

```diff
@@ -329,7 +329,7 @@ private SwitchToSlaveCopyThenBranch newSwitchToSlaveSpy() throws Exception
 
         StoreCopyClient storeCopyClient = mock( StoreCopyClient.class );
         Stream mockStream = mock( Stream.class );
-        when( mockStream.filter( any( Predicate.class ) ) ).thenReturn( mock( Stream.class ) );
+        when( mockStream.filter( any( Predicate.class ) ) ).thenReturn( mockStream);
```

Comment:
We are changing behaviour here slightly. Instead of returning a new mock we are now returning the same mock. Not sure if this is a problem but thought I's point it out.

Also, there is a missing space before the last parenthesis. Intellij auto-corrects it but checkstyle doesn't seem to catch it for some reason.

Reply:
Since these mocks are not configured/validated, it won't make any difference. They are basically just stubs.